internal/resources/resource_host.go

```diff
@@ -579,12 +559,28 @@ func resourceMetalHostUpdate(d *schema.ResourceData, meta interface{}) (err erro
 		}
 	}
 
+	// description
+	updDesc, ok := d.Get(hDescription).(string)
+	if ok && updDesc != host.Description {
+		host.Description = updDesc
```

Comment:
I'm wondering if the pattern of "updDesc != host.Description" does anything.  if not, less code if we just check ok

Reply:
Probably not. For update, TF does send values maintained in 'state' if they were unchanged in the updated plan.